app/controllers/billables_controller.rb

```diff
@@ -0,0 +1,5 @@
+class BillablesController < ApplicationController
+  def index
+    @projects = Project.all.billable
```

Comment:
You can (and should) actually call the billable scope directly on Project, so: `Project.billable` :)